server/plugin/configuration.go

```diff
@@ -24,18 +25,19 @@ func (p *MatterpollPlugin) OnConfigurationChange() error {
 		return errors.New("Empty trigger not allowed")
 	}
 
-	if oldConfiguration.Trigger != "" {
-		if err := p.API.UnregisterCommand("", oldConfiguration.Trigger); err != nil {
-			return errors.Wrap(err, "failed to unregister old command")
+	// Don't try to register a command if the plugin isn't activated
+	if p.isActivated() {
```

Comment:
I had to add this because `OnConfigurationChange()` is called before `OnActivate()`, but then `p.bundle` is `nil`.

Reply:
Can we call `initBundle` in `OnConfigurationChange`? If so, we can remove `if p.isActivated()`.

Reply:
Yes, we could do this. But it feels more hacky to me. 2/5

Reply:
As you said my suggestion looks hacky, but I don't have a better idea for now.
It's best not to call `OnConfigurationChange` before calling `OnActivate`, however it would be difficult.

So I'm fine to go with this code.